Detectors/TRD/macros/convertRun2ToRun3Digits.C

```diff
@@ -186,9 +188,19 @@ void convertRun2ToRun3Digits(TString qaOutPath = "",
         // cout << "det: " << det << " | " << "sector: " << sector << " | " << "stack: " << stack << " | " << "rows: " << nrows << endl;
 
         for (int row = 0; row < nrows; row++) {
-          for (int col = 0; col < AliTRDfeeParam::GetNcol(); col++) {
+          for (int col = 0; col < NCOLUMN; col++) {
+            int side = (col < NCOLUMN / 2) ? 0 : 1;
+            int rob = 2 * (row / NMCMROBINROW) + side;
+            int mcm = side == 0 ? (row * NMCMROBINROW + col / NCOLMCM) % NMCMROB : (row * NMCMROBINROW + (col - NCOLUMN / 2) / NCOLMCM) % NMCMROB;
+            int channel = 19 - (col % NCOLMCM);
             int tbsum = 0;
             ArrayADC adctimes;
+            bool isSharedRight = false, isSharedLeft = false;
+            if (col % NCOLMCM == 0 || col % NCOLMCM == 1) {
+              isSharedRight = true;
+            } else if (col % NCOLMCM == 17) {
+              isSharedLeft = true;
+            }
```

Comment:
I am not sure if I understand the code correctly, but it looks like you duplicate the entries from the run 2 digits manually. There should be a function, maybe in the AliTRDdigitsArray, that gives you access to the "hidden" ADC channels directly, without the need to duplicate.

Reply:
Yes, this is simply duplicating the digits manually. I can also check later on how to get the data properly. At the moment I am still struggling with the new tracklet format.

Reply:
This is still not done, but I would do it in a separate PR later, I think it is not strictly required